src/Microsoft.AspNetCore.Server.Kestrel/Internal/InternalKestrelServerOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Internal
+{
+    public class InternalKestrelServerOptions
```

Comment:
As small as it is, it should still have its unit test class. Add a unit test for the default value.